### ISSUE TITLE
Please add "-d" to your "inxi" command inside your "hw-probe" script to also output the optical drives info

### ISSUE THREAD

**Issue by sebma** [linuxhw/hw-probe#108]
Hi,
Your `inxi` commands inside your `hw-probe` Perl script are missing the output of the optical drives info : 
```perl
$ grep  -m1 "inxi " $(which hw-probe)
            $Inxi = runCmd("inxi -Fxxxzm --no-host 2>&1");
```
Can you please add `-d` to your `inxi` commands inside your `hw-probe` Perl script ?

**Comment by linuxhw** [comment]
Fixed by commit 5b449d0ead4297db0e12bb186b455e1f04f1e1a6

**Comment by sebma** [comment]
@linuxhw Thanks :+1: :smile:
